improver/utilities/cube_manipulation.py

```diff
@@ -712,7 +711,9 @@ def enforce_coordinate_ordering(cube, coord_names, anchor_start=True):
         new_dims = new_dims_end
 
     # transpose cube using new coordinate order
-    cube.transpose(new_dims)
+    if new_dims != list(range(len(new_dims))):
```

Comment:
👍

Reply:
sorted(new_dims) is faster for short lists... but this has a fixed cost (there's not much in it for short lists), so if, for whatever reason, we use a cube with a lot of dimensions, this would be the cheaper option

Reply:
> sorted(new_dims) is faster for short lists...

Interesting. Probably because there are less function calls involved. (`new_dims != sorted(new_dims)` is quite clear and easy to read.)

Reply:
Okay, I'll change it :-)